src/video/camerasource.cpp

```diff
@@ -337,7 +337,7 @@ void CameraSource::stream()
         }
 
       // Free the packet that was allocated by av_read_frame
-      av_free_packet(&packet);
+      av_packet_unref(&packet);
```

Comment:
are these new functions also present in older versions of ffmpeg? Because the oldest version we need to support is probably in Debian 8.